browser/ui/BUILD.gn

```diff
@@ -101,6 +101,8 @@ source_set("ui") {
     "webui/brave_webui_source.h",
     "webui/skus_internals_ui.cc",
     "webui/skus_internals_ui.h",
+    "webui/untrusted_sanitized_image_source.cc",
```

Comment:
can we put these in a separate target for `///brave/browser/ui:untrusted_sanitized_image_source`?

Reply:
actually maybe that doesn't help anything with the current state of things so it's ok for now